### DISCUSSION TITLE
weak network server.

### DISCUSSION THREAD

**Discussion by winter-warm-sun** [mitmproxy/mitmproxy#7026] [Q&A]
I want to build a weak network server. I have deployed the mitmproxy script on the centosLinux server and let the server and the local (win11) trust the certificate [the same version of the certificate], but why can't the server capture the https request? I hope you can help answer this question.

**Comment by emanuele-em** [comment]
Hi! Can you provide more details, from your description I can't understand how you run mitmproxy and which is the script for example. Maybe you are interested in reverse proxy mode, you can find the documentation here https://docs.mitmproxy.org/stable/concepts-modes/#reverse-proxy
